store/tikv/metrics.go

```diff
@@ -172,6 +172,14 @@ var (
 			Name:      "load_safepoint_total",
 			Help:      "Counter of load safepoint.",
 		}, []string{"type"})
+
+	secondaryLockCleanupTaskCounter = prometheus.NewCounterVec(
```

Comment:
s/secondaryLockCleanupTaskCounter/secondaryLockCleanupFailedCounter

Reply:
Maybe `secondaryLockCleanupFailureCounter` is better?